securesystemslib/storage.py

```diff
@@ -37,7 +38,8 @@ class StorageBackendInterface():
 
 
   @abc.abstractmethod
-  def get(self, filepath):
+  @contextmanager
+  def get(self, filepath: str) -> Generator[BinaryIO, None, None]:
```

Comment:
I think we can just use `Iterator[None]` for the return type here?
See https://stackoverflow.com/questions/49733699/python-type-hints-and-context-managers and https://github.com/python/typeshed/blob/master/stdlib/contextlib.pyi#L34

Reply:
if that works (and that would make sense) then I suppose it's `Iterator[BinaryIO]`?

Reply:
That would make more sense, indeed.